.env.local.example

```diff
@@ -0,0 +1,2 @@
+SENTRY_AUTH_TOKEN=
+NEXT_PUBLIC_SENTRY_DSN=
```

Comment:
Could you please update our [GitHub Actions deployment configuration](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/.github/workflows/deploy.yml) to ensure that the builds done on GitHub Actions are also using the correct environment variables?

You could use this [on push](https://github.com/kawalcovid19/wargabantuwarga.com/commit/5b6df79480e289c5b4d896f5e80305fe63f8c73f#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3) trigger to trigger the build based on pushes to our branch.

Reply:
So that means I've to change it like this?
```yml
on:
  push:
    branches: [feature/sentry-integration]
```

Reply:
After a second thought, I don't think that's a good idea, because that would eventually push your changes from this branch to production. 😅 
So if you want to do it, we need to create a separate CI script for the sake of testing & building only.
You can refer to [this tests.yml](https://github.com/ariya/dekontaminasi/blob/master/.github/workflows/tests.yml) as an example.